proxy/hdrs/MIME.h

```diff
@@ -987,6 +987,43 @@ struct MIMEFieldIter {
 class MIMEHdr : public HdrHeapSDKHandle
 {
 public:
+  /** Iterator over fields in the header.
+   * This iterator should be stable over field deletes, but not insertions.
+   */
+  class iterator
+  {
+    using self_type = iterator; ///< Self reference types.
+
+  public:
+    iterator() = default;
+
+    // STL iterator compliance types.
+    using difference_type   = void;
```

Comment:
Should this not be size_t ?

Reply:
No, because it's not possible to compute a difference.

Reply:
So, is there a way to run it with out deleting what's been inserted..? With out pulling it apart?